src/viz-sentence-population/VizSentencePopulation.js

```diff
@@ -0,0 +1,175 @@
+  record.district === district;
+
+function SentenceMap({ currentLocation, data, onLocationClick, width }) {
+  // eslint-disable-next-line react/prop-types
+  const DistrictLabel = ({ hover, locationId, topologyObjectId }) => {
+    const districtRecord = data.find(findDistrictRecord(topologyObjectId));
+    return districtRecord ? (
+      <ParticipantCount
+        active={locationId === districtRecord.district}
+        hover={hover}
+      >
+        {districtRecord.value}
+      </ParticipantCount>
+    ) : null;
+  };
+
+  const otherRecord = data.find(findDistrictRecord(OTHER));
+
+  return (
+    <SentenceMapWrapper>
+      <StateMap
+        aspectRatio={ASPECT_RATIO}
+        LabelComponent={DistrictLabel}
+        locationId={currentLocation}
+        onRegionClick={onLocationClick}
+        stateTopology={judicialDistrictsTopology}
+        width={width}
+      />
+      {otherRecord && (
+        <OtherWrapper
+          active={currentLocation === OTHER}
+          onClick={(e) => {
+            e.preventDefault();
+            onLocationClick(OTHER);
+          }}
+        >
+          <ParticipantCount as="div">{otherRecord.value}</ParticipantCount>
+          <OtherLabel>{OTHER_LABEL}</OtherLabel>
+        </OtherWrapper>
+      )}
+    </SentenceMapWrapper>
+  );
+}
+
+SentenceMap.propTypes = {
+  currentLocation: PropTypes.string.isRequired,
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      district: PropTypes.string.isRequired,
+      value: PropTypes.number.isRequired,
+    })
+  ).isRequired,
+  onLocationClick: PropTypes.func.isRequired,
+  width: PropTypes.number,
+};
+
+SentenceMap.defaultProps = {
+  width: 0,
+};
+
+export default function VizSentencePopulation(props) {
+  const {
+    data: { populationDemographics, locations },
+    locationId,
+    onLocationClick,
+  } = props;
+
+  const judicialDistrictTotals = populationDemographics
+    .filter(recordIsTotal)
+    .map((record) => {
+      const judicialDistrictData = [...locations, { id: TOTAL_KEY }].find(
+        // these are stored as both strings and numbers;
+        // doing an extra typecast here just to be safe
+        (loc) => `${loc.id}` === `${record.district}`
+      );
+      if (judicialDistrictData) {
+        return {
+          district: `${record.district}`,
+          value:
+            Number(record.incarceration_count) + Number(record.probation_count),
+        };
+      }
+      return null;
+    })
+    // drop any nulls from the previous step
+    .filter((record) => record);
+
+  return (
+    <PopulationViz
+      {...props}
+      MapComponent={SentenceMap}
+      mapComponentProps={{
+        data: judicialDistrictTotals,
+        currentLocation: locationId,
+        onLocationClick,
+      }}
+      mapLabel="Judicial districts in North Dakota"
+      locationAccessorFn={(record) => record.district}
+      populationAccessorFn={(record) =>
+        Number(record.incarceration_count) + Number(record.probation_count)
```

Comment:
I add the `incarceration_count` and `probation_count` such that the total number of sentences are displayed in this component.  I know we talked about adding a new control to allow users to filter based on sentence type, but that is a problem for another day.